app/config/services/services.xml

```diff
@@ -12,7 +12,12 @@
 			<tag name="controller.service_arguments" />
 		</prototype>
 
-		<service id="AppBundle\CitizenProject\CitizenProjectManager" />
+		<service id="AppBundle\CitizenProject\CitizenProjectManager">
```

Comment:
Forgot to revert that line ?

Reply:
No, without this I have an error `Circular reference detected for service "app.glide", path: "app.glide".`
I did not find the reason...